src/components/Navigation/Navigation.jsx

```diff
@@ -8,6 +8,7 @@ import { upperFirst } from 'lodash';
 import axios from 'axios';
 import cx from 'classnames';
 
+import { useScrollDirection } from '../../custom-hooks/useScrollDirection';
```

Comment:
```suggestion
import { useScrollDirection } from '../../hooks/useScrollDirection';
```

And also, please create `index.js` file to reexport hooks so we can write in the way bellow

```suggestion
import { useScrollDirection } from '../../hooks';
```

Reply:
fixed